its/src/test/java/com/sonar/maven/it/suite/MavenTest.java

```diff
@@ -41,7 +39,6 @@
 class MavenTest extends AbstractMavenTest {
 
   private static final String MODULE_START_7_6 = "------------- Run sensors on module ";
```

Comment:
I guess we can simply rename this constant if it is the only one left
```suggestion
  private static final String MODULE_START = "------------- Run sensors on module ";
```